src/sub/app.py

```diff
@@ -22,6 +22,23 @@
 
 logger = get_logger()
 
+
+def is_container() -> bool:
+    import requests
+
+    try:
+        requests.get(f"http://dynalite:{CFG.DYNALITE_PORT}")
+        return True
+    except Exception as e:
+        pass
+    return False
+
+
+if is_container():
+    stripe.log = "debug"
```

Comment:
does case matter? i would think DEBUG would make more sense; if it doesnt mater, consider changing it to uppercase

Reply:
It is case insensitive.